client/src/components/Modal/InnerModal.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const InnerModal = ({ title, description, imageSrc }) => {
+  return (
+    <div>
+      <img src={imageSrc} alt="" />
```

Comment:
always offer the alt tag with content, if it isn't just decoration, background noise or didnt transport any meaning
```suggestion
      <img src={imageSrc} alt="image description or content" />
```

Reply:
Thanks, I put in the title now, that I get back from API.